docs/source.rst

```diff
@@ -7,36 +7,26 @@ Source Guide
           computer should you use? what network should you be on? etc.
 
 
-Choose who to submit to
------------------------
-
-Each SecureDrop instance is totally independent, and submissions to that instance are only available to journalists associated with that organization.
-
-All organizations have a *landing page* that provides their own organization-specific recommendations for using SecureDrop. We encourage you to consider an organization's landing page before submitting to them.
-
-Most organizations make their landing page prominently accessible from their main website's homepage. You can also find an incomplete list of organizations accepting submissions through SecureDrop on the `SecureDrop Directory`_ maintained by Freedom of the Press Foundation.
-
-.. _`SecureDrop Directory`: https://securedrop.org/directory
+Choosing the right location
+---------------------------
 
+When national security is involved, we suggest you buy a new computer,
+a USB key and pay cash. In any case you must then find a busy
+coworking place or cyber café you don't regularly go to and seat at a
+place with your back to a wall to avoid cameras capturing information
+on your screen or keystrokes.
```

Comment:
Nits:

* `s/café/cafe/` (in ef1cdb087 we used "cafe" and consistency is grand)
* `s/seat/sit/`